hw/ip/otbn/data/bignum-insns.yml

```diff
@@ -313,17 +321,26 @@
     - *mulqacc-wrs2
     - *mulqacc-wrs2-qwsel
     - *mulqacc-acc-shift-imm
+    - *bn-flag-group-operand
   syntax: |
     [<zero_acc>] <wrd>.<wrd_hwsel>,
-    <wrs1>.<wrs1_qwsel>, <wrs2>.<wrs2_qwsel>, <acc_shift_imm>
+    <wrs1>.<wrs1_qwsel>, <wrs2>.<wrs2_qwsel>, <acc_shift_imm>[, FG<flag_group>]
   glued-ops: true
   doc: |
-    Multiplies two `WLEN/4` WDR values, shifts the product by `<acc_shift_imm>` and adds the result to the accumulator.
-    Next, shifts the resulting accumulator right by half a word.
-    The bits that are shifted out are written to a half-word of `<wrd>`, selected with `<wrd_hwsel>`.
-
+    Multiplies two `WLEN/4` WDR values, shifts the product by `acc_shift_imm` bits and adds the result to the accumulator.
+    Next, shifts the resulting accumulator right by half a word (128 bits).
+    The bits that are shifted out are written to a half-word of `wrd`, selected with `wrd_hwsel`.
+
+    This instruction has slightly complicated rules for how it updates the flags in `flag_group`.
```

Comment:
Let's just state the facts and remove the "slightly complicated" wording.

Reply:
Good point. I probably wrote that sentence after drafting the stuff that followed for the third time :-)